In [10]:
import pandas as pd

file_path = 'Table_E1c.csv'
data = pd.read_csv(file_path, header=None, encoding='latin1')

years = [int(year.split()[1]) for year in data.iloc[2, 1:].values]
receipts = data.iloc[4, 1:].str.replace(',', '').astype(float)
total_resolutions = data.iloc[5, 1:].str.replace(',', '').astype(float)
settlements = data.iloc[7, 1:].str.replace(',', '').astype(float)
withdrawals_with_benefits = data.iloc[10, 1:].str.replace(',', '').astype(float)
administrative_closures = data.iloc[10, 1:].str.replace(',', '').astype(float)
no_reasonable_cause = data.iloc[16, 1:].str.replace(',', '').astype(float)
reasonable_cause = data.iloc[19, 1:].str.replace(',', '').astype(float)
successful_conciliations = data.iloc[22, 1:].str.replace(',', '').astype(float)
unsuccessful_conciliations = data.iloc[26, 1:].str.replace(',', '').astype(float)
merit_resolution = data.iloc[29, 1:].str.replace(',', '').astype(float)

df = pd.DataFrame({
    'Year': years,
    'Receipts': receipts,
    'Total Resolutions': total_resolutions,
    'Settlements': settlements,
    'Withdrawals with Benefits': withdrawals_with_benefits,
    'Administrative Closures': administrative_closures,
    'No Reasonable Cause': no_reasonable_cause,
    'Reasonable Cause': reasonable_cause,
    'Successful Conciliations': successful_conciliations,
    'Unsuccessful Conciliations': unsuccessful_conciliations,
    'Merit Resolution': merit_resolution
})

df_2009_2017 = df[(df['Year'] >= 2009) & (df['Year'] <= 2017)].copy()
df_2009_2017.loc[df_2009_2017['Year'] == 2017, df.columns[1:]] /= 2

means = df_2009_2017.mean().round(2)
totals = df_2009_2017.sum()
percentages = (totals / totals['Total Resolutions'] * 100).round(2)

def print_table(title, data):
    print(f"\n{title}")
    print("-" * (len(title) + 2))
    for key, value in data.items():
        print(f"{key}: {value:.2f}")

print_table("Means from 2009 to Q2 2017", means)
print_table("Percentages from 2009 to Q2 2017", percentages)


Means from 2009 to Q2 2017:
Year                           2013.000000
Receipts                      88675.333333
Total Resolutions             93216.611111
Settlements                    8088.611111
Withdrawals with Benefits      5064.888889
Administrative Closures        5064.888889
No Reasonable Cause           61360.055556
Reasonable Cause               3497.944444
Successful Conciliations       1262.777778
Unsuccessful Conciliations     2235.166667
Merit Resolution              16651.444444
dtype: float64

Percentages from 2009 to Q2 2017:
Year                            2.159486
Receipts                       95.128253
Total Resolutions             100.000000
Settlements                     8.677221
Withdrawals with Benefits       5.433462
Administrative Closures         5.433462
No Reasonable Cause            65.825237
Reasonable Cause                3.752490
Successful Conciliations        1.354670
Unsuccessful Conciliations      2.397820
Merit Resolution               17.86317

Data source:
https://www.eeoc.gov/data/enforcement-and-litigation-statistics-0

Based on cases filed between January 1, 2009 through June 30, 2017, discrimination lawsuits are by far the most common (87% of cases), followed by retaliation (66%) and harassment (35%).  However, employment cases often involve overlapping kinds of claims – discrimination and retaliation claims are combined more than half the time (54%), and the other two combinations occur in about a third of cases.  Cases with all three tags comprise just under a quarter of the cases (24%).

employment_tags

Number of cases tagged as Discrimination, Harassment, and/or Retaliation with overlaps (cases filed 2009 through 2017 Q2)

These tags help practitioners focus in on the relevant cases, ensuring that other statistics and insights are more precise that would otherwise be possible.  

https://lexmachina.com/blog/lex-machina-launches-legal-analytics-for-employment/